test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -0,0 +1,241 @@
+			name: 'Block: Cover',
+		} );
+
+		await coverBlock
+			.getByTestId( 'form-file-upload-input' )
+			.setInputFiles( testImage );
+
+		const [ backgroundDimColor, backgroundDimOpacity ] =
+			await getBackgroundColorAndOpacity(
+				coverBlock.locator( 'span[aria-hidden="true"]' )
+			);
+		expect( backgroundDimColor ).toBe( 'rgb(0, 0, 0)' );
+		expect( backgroundDimOpacity ).toBe( '0.5' );
+	} );
+
+	test( 'can have the title edited', async ( { page, editor } ) => {
+		const titleText = 'foo';
+
+		await editor.insertBlock( { name: 'core/cover' } );
+
+		// Choose a color swatch to transform the placeholder block into
+		// a functioning block.
+		await page
+			.getByRole( 'button', {
+				name: 'Color: Black',
+			} )
+			.click();
+
+		// Activate the paragraph block inside the Cover block.
+		// The name of the block differs depending on whether text has been entered or not.
+		const coverParagraphLocator = page.getByRole( 'document', {
+			name: /Paragraph block|Empty block; start writing or type forward slash to choose a block/,
+		} );
+		await expect( coverParagraphLocator ).toBeEditable();
+
+		await coverParagraphLocator.fill( titleText );
+
+		await expect( coverParagraphLocator ).toContainText( titleText );
+	} );
+
+	test( 'can be resized using drag & drop', async ( { page, editor } ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+
+		// Open the document sidebar.
+		await editor.openDocumentSettingsSidebar();
+
+		await page
+			.getByRole( 'button', {
+				name: 'Color: Black',
+			} )
+			.click();
+
+		// Open the block list viewer from the toolbar.
+		await page
+			.getByRole( 'toolbar', { name: 'Document tools' } )
+			.getByRole( 'button', { name: 'Document Overview' } )
+			.click();
+
+		// Select the Cover block from the docuemnt overview.
+		await page.getByRole( 'link', { name: 'Cover' } ).click();
+
+		// Ensure there the default value for the minimum height of cover is undefined.
+		await page.getByRole( 'tab', { name: 'Styles' } ).click();
+		const defaultHeightValue = await page
+			.getByLabel( 'Minimum height of cover' )
+			.inputValue();
+		expect( defaultHeightValue ).toBeFalsy();
+
+		// Establish locators for the Cover block and the resize handler.
+		const coverBlock = page.getByRole( 'document', {
+			name: 'Block: Cover',
+		} );
+		const coverBlockResizeHandle = page.locator(
+			'.components-resizable-box__handle-bottom'
+		);
+
+		// Establish the existing bounding boxes for the Cover block.
+		const coverBlockBox = await coverBlock.boundingBox();
+
+		expect( coverBlockBox.height ).toEqual( 450 );
+
+		// Resize the block by 100px.
+		await coverBlockResizeHandle.hover();
+		await page.mouse.down();
+		await page.mouse.move(
+			coverBlockBox.x,
+			coverBlockBox.y + coverBlockBox.height + 100,
+			{ steps: 5 }
+		);
+		await page.mouse.up();
+
+		const newCoverBlockBox = await coverBlock.boundingBox();
+		expect( newCoverBlockBox.height ).toBeGreaterThanOrEqual(
```

Comment:
Just being curious, why couldn't we just use `toBe` here? Sub-pixel issue?

Reply:
The boundingbox size that's returned _after_the resize is 562, which is not exactly 100 more than the original size. The original Puppeteer spec tested that the value changed by more than 100 and so I applied a similar line of thinking in migrating the test.

With that said, I went digging to explain the discrepancy.

The first step was to output the bounding box for all involved elements.
Resize handle  { x: 519, y: 683.5, width: 472, height: 23 }
Cover block { x: 519, y: 245.21875, width: 472, height: 450 }

Notice that the coverBlock.height + ( resizeHandle.height / 2 ) + 100 = 561.5 which likely rounds up to 562.

Checking the specific pixel value would obviously be testing an implementation detail. 
In similar situations in the past I would have stuck with either the current implementation, or go with the Puppeteer spec approach of checking that the bounding box changed by at least 100px. What do you think?

Reply:
Thanks for digging into the details! I appreciate it a lot!

Would an API like [`toBeCloseTo`](https://jestjs.io/docs/expect#tobeclosetonumber-numdigits) help in this case? (I'm not sure if it's available in Playwright yet though!)